src/main/java/com/kop/daegudot/web/dto/OpendataDto.java

```diff
@@ -0,0 +1,31 @@
+package com.kop.daegudot.web.dto;
+
+import com.kop.daegudot.domain.opendatas.Opendatas;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class OpendataDto {
+    private String address;
+    private String attractContents;
+    private String attractName;
+    private String homepage;
+    private String telephone;
+
+    @Builder
+    public OpendataDto(String address, String attractContents, String attractName, String homepage,
+                       String telephone){
+        this.address = address;
+        this.attractContents = attractContents;
+        this.attractName = attractName;
+        this.homepage = homepage;
+        this.telephone = telephone;
+    }
+
+    public Opendatas toEntity() { return Opendatas.builder().address(address)
+        .attractContents(attractContents).attractName(attractName).homepage(homepage)
+            .telephone(telephone).build();

```

Comment:
```suggestion
    public Opendatas toEntity() { 
        return Opendatas.builder().address(address)
            .attractContents(attractContents).attractName(attractName).homepage(homepage)
            .telephone(telephone).build();

```

return 구문이 아래로 내려가게 수정해주세요.